leakcanary-support-fragment/src/androidTest/java/com/squareup/leakcanary/tests/FragmentLeakTest.java

```diff
@@ -46,4 +59,27 @@ public void fragmentShouldLeak() throws InterruptedException {
       throw new AssertionError("Expected a leak of TestFragment, not " + leakingClassName);
     }
   }
+
+  @Test
+  public void fragmentViewShouldLeak() {
+    activityRule.launchActivity(new Intent(InstrumentationRegistry.getTargetContext(), NastyActivity.class));
+
+    Espresso.onView(ViewMatchers.withId(R.id.add)).perform(click());
+    Espresso.onView(ViewMatchers.withId(R.id.add)).perform(click());
```

Comment:
Why is the test adding the fragment twice?

Reply:
could you please explain why now after version 1.6.2 I got leak on replace + addToBackStack operations, and how to handle it?

Reply:
Is this change necessary? Checking leaks onDestroyView introduced a bunch of leaks in our app. Sure they are small amounts of memory that we could have deallocated. To resolve the leaks, we set views to null during onDestroyView, which introduced potential NPEs (imagine a view used in a callback which happens to get called after onDestroyView), which are much worse than the tiny memory leaks IMO.